File: C_string/rs/src/c_string/c1.rs
// Tulis function getStringInfo di sini
#[derive(Debug)]
pub struct InfoString {
    pub length: usize,
    pub first: char,
    pub last: char,
}

pub fn get_string_info(text: &str) -> InfoString {
    let length = text.chars().count(); // panjang string
    let first = text.chars().next().unwrap_or('\0'); // karakter pertama, '\0' kalau kosong
    let last = text.chars().last().unwrap_or('\0'); // karakter terakhir, '\0' kalau kosong

    InfoString {
        length,
        first,
        last,
    }
}

File: projects/experimental/challenge/C_string/js/c1.js
// Tulis function getStringInfo di sini
function getStringInfo(text) {
    return {
        length: text.length,
        first: text.charAt(0),
        last: text.charAt(text.length - 1)
    }
}
console.log(getStringInfo("Hello"));
console.log(getStringInfo("JavaScript"));
console.log(getStringInfo("A"));